src/hooks/protections/received_event.cpp

```diff
@@ -564,6 +564,11 @@ namespace big
 				g_pointers->m_gta.m_send_event_ack(event_manager, source_player, target_player, event_index, event_handled_bitset);
 				return;
 			}
+			else if (type == WorldStateDataType::PopMultiplierArea && g.protections.stop_traffic) // never used for legitimate reasons

```

Comment:
Not even in races where traffic may end up being modified?

Reply:
Races usually happen on areas that are like under the world like -8000 X -8000 Y or something like that IIRC.

Reply:
This is used legitimately.
![image](https://github.com/YimMenu/YimMenu/assets/45642596/15eb1770-3082-4f25-b978-b8cfe325e3c2)

Reply:
> Races usually happen on areas that are like under the world like -8000 X -8000 Y or something like that IIRC.

It modifies the entire map at times.
![image](https://github.com/YimMenu/YimMenu/assets/45642596/628d4a74-5b4a-481a-9feb-4471b177356c)

Reply:
@SkiddyToast Do you know what circumstances the freemode script calls this function?